### ISSUE TITLE
Installation post - mini-test becoming part of official

### ISSUE THREAD

**Issue by Mirv** [kematzy/minitest-have_tag#1]
Might just be how new I am, but I read the documentation ...

1) installed gem file
2) verified it appeared in the gemfile.lock
3) next instruction calls for inserting `require 'minitest/have_tag'` in the `spec_helper.rb` ...which doesn't exist in my ruby on rails despite having the minitest.  

I tried making a blank spec_helper.rb in the tests/helpers & included it in the `test_helper.rb`.

Then I moved the `require 'minitest/have_tag'` in to the `test_helper.rb` directly.

I'm getting:

`rake aborted!
NoMethodError: undefined method `infect_an_assertion' for Minitest::Expectations:Module`

Anyone have hints on this one?

**Comment by kematzy** [comment]
Mirv,

I don't use Rails and have therefore not tested with Rails. Nor do I have any great clues about how to implement this in Rails land.

I think your problem is related gem / code loading order, as the `infect_an_assertion` method is defined in the main `minitest` gem.  

So try changing your load order, and it might help.

**Comment by Mirv** [comment]
Awesome thank you!  Will update shortly.

I tried 5 things, watched 2 videos & with my cloud9 env, I'm just not equipped to trouble shoot the install since minitest was pulled into rails officially.  It might be really simple, but cost/time ratio isn't worth it when I can just count the content instead.

Thanks for the suggestion - it looks like it is the right path to pursue, the gem file installs & shows up in there...

Funny story, the predecessor `infect_with_assertions has been removed due to excessive clever. Use infect_an_assertion directly instead.`   [Link](http://apidock.com/ruby/Module/infect_with_assertions)

I'm closing this one out, happy new years!
